core/controllers/learner_dashboard.py

```diff
@@ -122,12 +139,32 @@ def get(self, exploration_id, thread_id):
             self.user_id, exploration_id, thread_id, message_ids)
 
         message_summary_list = []
+        suggestion = feedback_services.get_suggestion(exploration_id, thread_id)
+
+        if suggestion:
+            exploration = exp_services.get_exploration_by_id(exploration_id)
+            current_content_html = (
+                exploration.states[
+                    suggestion.state_name].content.to_dict()['html'])
```

Comment:
Seems a bit indirect; couldn't we do something like content.get_html() or content.html or something like that? Not sure why converting to a dict is necessary.